membership-attribute-service/app/services/ScanamoAttributeService.scala

```diff
@@ -1,6 +1,6 @@
 package services
 
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsync
```

Comment:
I changed this to use the interface not the concrete class. This is in preparation for switching to use the builder as instantiating a AmazonWebServiceClient alone is now deprecated.